Backpack/src/main/java/net/skyscanner/backpack/util/BpkTheme.kt

```diff
@@ -46,6 +46,7 @@ class BpkTheme {
      */
     @JvmStatic
     @ColorInt
+    @Deprecated("Use bpkCoreAccent instead")
```

Comment:
This is no longer really helpful since we moved to the new theming approach in the app